app/models/enterprise.rb

```diff
@@ -30,6 +30,7 @@ class Enterprise < ApplicationRecord
   preference :shopfront_product_sorting_method, :string, default: "by_category"
   preference :invoice_order_by_supplier, :boolean, default: false
   preference :product_low_stock_display, :boolean, default: false
+  preference :hide_ofn_navigation, :boolean, default: false
```

Comment:
This is the easiest option to add a field but I would prefer a change in the `enterprises` table. It's easier to change or clean up later. We also avoid the `preferred_` prefix in the name and don't need an additional query to fetch this option.

Is that just my opinion? @dacook

Reply:
I'm not sure. I think for things that are not part of the model's core functionality, it makes sense to separate them out and not load them everytime you load the record (unnecessary memory and processing to instantiate the model, however small).

But spree's `preferred` feature has it's own drawbacks as you mentioned.

So I'm not really sure. But it's worth pointing out that in the following PR, we do add an image attachment that shows/hides based on this value. So maybe it is worth storing on the enterprises table?

Reply:
I've create a new column: edb72fb833